sdk/README.md

```diff
@@ -12,15 +12,22 @@
 ![Jenkins Coverage](https://img.shields.io/jenkins/coverage/cobertura?jobUrl=https%3A%2F%2Fjenkins.lisk.com%2Fjob%2Flisk-sdk%2Fjob%2Fdevelopment)
 [![License: Apache 2.0](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 
-## Beta phase
+## Sapphire phase [<img src="https://lisk.com/sites/default/files/2021-05/ico-sapphire.png" style="width:24px;vertical-align:middle">](https://lisk.com/roadmap)
 
-Please read this carefully. With this repository we have opened access to the beta release of the Lisk SDK. We have released the Lisk SDK in its current form in order for us to improve the development experience through community feedback and contributions.
+We have completed [Emerald phase](https://lisk.com/roadmap)[<img src="https://lisk.com/sites/default/files/2021-05/ico-emerald.png" style="width:20px;vertical-align:middle">](https://lisk.com/roadmap) that encapsulates improvement in the key areas of network security and reliability, economics, consensus, and longevity, whereby a new address system, a new fee system, and upgrading the consensus algorithm to an improved version of DPoS were successfully completed. During the Emerald phase all protocol improvements in form of 36 Lisk Improvement Proposals (LIPs) were researched, published and are implemented in Lisk SDK.
 
-We strictly discourage anyone from using the beta release of the Lisk SDK for any production-based blockchain applications, i.e. a blockchain operating on a real mainnet. Throughout the beta phase, there will be significant changes in the Lisk protocol and implementation, which will eventually bring accessibility and reliability to a level that is feasible for production-based blockchain applications. At this time we only recommend the Lisk SDK for proof-of-concept blockchain applications, i.e. a blockchain operating on a fake testnet.
+We are currently working on [Sapphire phase](https://lisk.com/roadmap)[<img src="https://lisk.com/sites/default/files/2021-05/ico-sapphire.png" style="width:20px;vertical-align:middle">](https://lisk.com/roadmap) that brings the [Lisk interoperability solution blog](https://lisk.com/blog/research/lisk-interoperability-solution-published) which is fully researched and all Lisk Improvement Proposals (LIPs) are published and currently being actively developed. It consists of 8 roadmap objectives,
```

Comment:
```suggestion
We are currently working on [Sapphire phase](https://lisk.com/roadmap)[<img src="https://lisk.com/sites/default/files/2021-05/ico-sapphire.png" style="width:20px;vertical-align:middle">](https://lisk.com/roadmap) which is summarized in the [Lisk interoperability solution blog.](https://lisk.com/blog/research/lisk-interoperability-solution-published) All related Lisk Improvement Proposals (LIPs) are published and currently being actively developed. It consists of 8 roadmap objectives.
```

Reply:
sapphire phase is not summarised in Lisk interoperability solution blog but it is part of sapphire phase. Adding the rest of the suggested code in separate commit

Reply:
Ok the sentence just reads strange. How does a phase "bring a blog", not sure if it is grammatically or semantically correct.

Reply:
Like i said, the sentence is partially updated, please check the commit, there is no "blog"

Reply:
I see that you fixed it now. Good work!

Reply:
Actually it still reads strangely. Instead of the word "brings" I suggest "implements" or another word.

Reply:
since it is researched, designed and implemented, i thought it will be better to use "brings", I'll update it anyways